<launch>
    <arg name="uav_id" default="0"/>
    <arg name="namespace" value="$(eval 'uav' + str(arg('uav_id')))"/>
    <node pkg="px4_application" type="form_publisher" name="form_publisher" output="screen" ns="$(arg namespace)">
        <rosparam command="load" file="$(find px4_application)/config/rate_config/spin_rate_config.yaml" />
	</node>
        
</launch>